drivers/spi/spi_ll_stm32.h

```diff
@@ -152,10 +184,11 @@ static inline void ll_func_disable_int_errors(SPI_TypeDef *spi)
 static inline uint32_t ll_func_spi_is_busy(SPI_TypeDef *spi)
 {
 #if defined(CONFIG_SOC_SERIES_STM32MP1X) || \
-	defined(CONFIG_SOC_SERIES_STM32H7X) || \
 	defined(CONFIG_SOC_SERIES_STM32U5X)
 	return (!LL_SPI_IsActiveFlag_MODF(spi) &&
 		!LL_SPI_IsActiveFlag_TXC(spi));
+#elif defined(CONFIG_SOC_SERIES_STM32H7X)
+	return LL_SPI_IsActiveFlag_EOT(spi);
```

Comment:
You said IP is the same for H7 and U5, so is it normal to have here a different ?

Reply:
This change is now tested in SPI with INTerrupt  mode